mk/spksrc.cross-env.mk

```diff
@@ -37,6 +37,7 @@ ENV += TC=$(TC)
 ENV += $(TC_ENV)
 endif
 
-#ifneq ($(COMPILE_MAKE_OPTIONS),)
-#ENV += MAKEFLAGS="$(COMPILE_MAKE_OPTIONS)"
-#endif
+# Set parallel options in caller
+ifeq ($(PARALLEL_MAKE),max)
+MAKEFLAGS += -j$(shell nproc)
```

Comment:
What happens when PARALLEL_MAKE is a number? IIRC PARALLEL_MAKE was used to limit how many cores to use. I guess it doesn't actually matter, since we can just limit docker to the number of cores we want to give it. Any thoughts @hgy59? If the new behaviour stays, rather than max, true/false would make more sense then.

Reply:
```
$ time PARALLEL_MAKE=3 make -C spk/fish arch-x64-7.0

real    3m8.738s
user    2m44.456s
sys     0m24.859s
```